tests/integration/test_smart_contracts.py

```diff
@@ -147,6 +147,11 @@ def test_get_contract_with_contract_id(self):
         self.assertEqual(response.get("status"), 200, response)
         jsonschema.validate(response.get("response"), schema.smart_contract_at_rest_schema)
 
+    def test_get_contract_logs_with_contract_id(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID)
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
```

Comment:
We always try to check the shape of the response with json schemas when possible. Just returning 200 doesn't mean we're getting the correct data back